package-lock.json

```diff
@@ -3612,7 +3612,7 @@
       "optional": true
     },
     "upgrade_types": {
-      "version": "github:CarnegieLearningWeb/educational-experiment-types#7726de2644d4006ea4b5ddfb7edc17cbc138674b",
+      "version": "github:CarnegieLearningWeb/educational-experiment-types#67359be6e740d73575ce752810a335966eb6198e",
```

Comment:
This file isn't Java-related, not sure why it's being touched in this PR ... ?

Reply:
Yeah! i pulled this to see some of the interfaces in JS Client lib to make sure they are up to date with JAVA client lib. it was pushed by mistake in this commit. It should not be here.